src/assets/custom/css/_sass/_services-page-intro.scss

```
@@ -37,6 +37,22 @@
   }
 }
 
+.services-page-intro__title-wrapper {
+  display: flex;
+  align-items: center;
+  @include small {
+    margin-bottom: 30px;
+  }
+  @include medium {
+    margin-bottom: 40px;
+  }
+  @include large-and-extra-large{
+    margin-bottom: 100px;
+  }
+  justify-content: space-between;
+  flex-wrap: wrap;
+}
```

Can we bring those last two rules up to live with the rest of the rules being applied to the generic

```suggestion
.services-page-intro__title-wrapper {
  align-items: center;
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  
  @include small {
    margin-bottom: 30px;
  }
  @include medium {
    margin-bottom: 40px;
  }
  @include large-and-extra-large{
    margin-bottom: 100px;
  }
}
```

---

Fixed in this commit: [9673859](https://github.com/codurance/site/pull/1875/commits/9673859335ede632b6ab845ba3be8777f5cdae14)